src/toolkits/object_detection/object_detector.cpp

```diff
@@ -375,15 +389,15 @@ variant_map_type object_detector::perform_evaluation(gl_sframe data,
   int num_outputs_per_anchor = 5 + static_cast<int>(class_labels.size());
   int num_output_channels = num_outputs_per_anchor * anchor_boxes().size();
   std::unique_ptr<model_backend> model = ctx->create_object_detector(
-      /* n */     options.value("batch_size"),
-      /* c_in */  NUM_INPUT_CHANNELS,
-      /* h_in */  GRID_SIZE * SPATIAL_REDUCTION,
-      /* w_in */  GRID_SIZE * SPATIAL_REDUCTION,
-      /* c_out */ num_output_channels,
-      /* h_out */ GRID_SIZE,
-      /* w_out */ GRID_SIZE,
-      get_prediction_config(),
-      get_model_params());
+      /* n       */ options.value("batch_size"),
```

Comment:
clang-format wanted to remove the extra white space between the comment and the code, so I just moved the formatting white space into the comment....